lnd.go

```diff
@@ -938,6 +943,10 @@ type WalletUnlockParams struct {
 	// ChansToRestore a set of static channel backups that should be
 	// restored before the main server instance starts up.
 	ChansToRestore walletunlocker.ChannelsToRecover
+
+	// Loader is a reference to the wallet loader, which should be unloaded
+	// by UnloadWallet on shutdown
+	Loader *wallet.Loader
```

Comment:
instead of passing a `Loader` reference here, what about making it a `UnloadWallet func() error`?

Reply:
Oh yeah passing a function pointer seems more appropriate than dragging out the whole Loader instance.

Reply:
Fixed